src/components/input/Select.tsx

```diff
@@ -405,6 +433,19 @@ type BaseSelectProps = CompositeProps & {
 
   /** A callback passed to the listbox onScroll */
   onListboxScroll?: JSX.HTMLAttributes<HTMLUListElement>['onScroll'];
+
+  /**
+   * Indicates how overflowing content should be handled in the listbox.
+   * * `truncate` -> Truncate the options so that they all fit in one line. An
+   *                 ellipsis will be rendered where needed.
+   *                 This is the default value.
+   * * `wrap` -> Let options content wrap multiple lines so that all content is
+   *             visible.
+   *
+   * This behavior can be also overwritten by providing more complex options
+   * content.
```

Comment:
I don't find the meaning of this very clear. I presume what you mean to say is that when the rendered content meets certain criteria, the `wrap` and `truncate` styles applied to the listbox won't have an effect and the caller will have to provide their own styles to control this.

One way to resolve this would be to document the behavior explicitly as setting the `text-overflow` and `white-space` CSS properties and noting as a caveat that some content may need to provide its own styling to handle cases where the content is not wide enough.

Reply:
> I don't find the meaning of this very clear. I presume what you mean to say is that when the rendered content meets certain criteria, the `wrap` and `truncate` styles applied to the listbox won't have an effect and the caller will have to provide their own styles to control this.

Yes, that's correct.

> One way to resolve this would be to document the behavior explicitly as setting the `text-overflow` and `white-space` CSS properties and noting as a caveat that some content may need to provide its own styling to handle cases where the content is not wide enough.

Ok!